docs/README.md

```diff
@@ -27,10 +34,14 @@ export PATH=$PATH:/Library/TeX/Distributions/.DefaultTeX/Contents/Programs/texbi
 
 ### Building Docs:
 ```bash
-python setup.py build_ext --inplace
 cd docs
 git submodule update --init
-./build.sh # or ./build-public.sh when deploying to aihabitat.org
+
+cd habitat-sim
+./build.sh
```

Comment:
You may need `--with-bullet` to compile the physics libraries in order for their docstrings to be available after building the docs

Reply:
That might be it. My method somehow make the "magnum" part load, maybe that was the physics libraries.

Reply:
Magnum will load either way, but some parts of the code are compiled out when bullet isn't included.